airbyte-webapp/src/components/ui/Switch/Switch.tsx

```diff
@@ -41,7 +43,7 @@ export const Switch: React.FC<SwitchProps> = ({
   });
 
   return (
-    <label className={labelStyle}>
+    <label className={labelStyle} {...(testId ? { "data-testid": `${testId}-switch` } : {})}>
```

Comment:
It should not append `-switch`... it should respect what was set out.

Reply:
Will fix 👍